.github/workflows/maintainer_merge_comment.yml

```
@@ -28,7 +28,7 @@ jobs:
         run: |
           ./scripts/get_tlabel.sh "${PR}" >> "$GITHUB_OUTPUT"
         env:
-          PR:  /repos/{owner}/{repo}/issues/{pull_number}
+          PR:  /repos/leanprover-community/mathlib4/issues/${{ github.event.pull_request.number }}
```

This workflow is triggered by the `issue_comment` event so we need to get the PR number this way 🤷 
```suggestion
          PR:  /repos/leanprover-community/mathlib4/issues/${{ github.event.issue.number }}
```

---

There are *so* many pitfalls!  Thanks for helping me navigate them!  :smile: